composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "brick/math": "^0.8.8 || ^0.9 || ^0.10 || ^0.11",
+        "brick/math": "^0.8.8 || ^0.9 || ^0.10 || ^0.11 || ^0.12",
```

Comment:
FYI: I tried installing brick/math with --prefer-lowest and run a static analysis, but it failed since BigInteger::toBytes does not exist in v0.8.8. This is out of scope for this PR, so I ignored.